src/components/LoginButton/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { Button } from "@sendgrid/ui-components/button";
+
+const LoginButton = ({ handleLoginSubmit }) => (
```

Comment:
Yeah same thing as before since the Loading/Login Buttons and Alerts are all specific to the Login page it's better to keep them all local in the same components/Login/index.js and declared as SFCs above so you don't have to open so many different files!